src/tribler/gui/tribler_app.py

```diff
@@ -32,6 +33,9 @@ def on_app_message(self, msg):
             self.handle_uri(msg)
 
     def handle_uri(self, uri):
+        if not self.activation_window():
```

Comment:
**nitpick:** I think we can store `activation_window` in a local variable to not repeat `self.activation_window()` call multiple times. This way, the code will be a bit more concise. If I understand correctly, `activation_window` is the instance of `TriblerWindow`, so we can name the variable `tribler_window` to make it clearer.

Reply:
Agree, I'll do the refactoring in a separate PR.